tests/LANraragi/Plugin/Metadata/ChaikaFile.t

```diff
@@ -15,6 +15,7 @@ use Test::Deep;
 my $cwd     = getcwd();
 my $SAMPLES = "$cwd/tests/samples";
 require "$cwd/tests/mocks.pl";
+setup_redis_mock();
```

Comment:
The plugin tests don't really need the redis mocks, right? It doesn't hurt to add them I guess